centml/compile.py

```diff
@@ -0,0 +1,76 @@
+import builtins
+from typing import Callable, Dict, List, Optional, Union
+
+import torch
+from prometheus_client import start_http_server
+from torch._subclasses.fake_tensor import FakeTensorMode
+
+from centml.compiler.backend import centml_dynamo_backend
+from centml.compiler.config import OperationMode, settings
+from centml.compiler.metrics import A10_time_metric
+from centml.compiler.profiler import Profiler
+
+start_http_server(8000)
+A10_time = 0
```

Comment:
we should try to make the implementation a bit more generic than this. Create an environment variable to accept a list of GPU types. You can set the default value as A10 for now. Then you can write an implementation that exports metrics for all the GPU types in that list.